inductiva/molecules/scenarios/protein_solvation/protein_solvation.py

```diff
@@ -51,7 +61,8 @@ def simulate(
             simulation_time_ns: float = 10,  # ns
             output_timestep_ps: float = 1,  # ps
             integrator: Literal["md", "sd", "bd"] = "md",
-            n_steps_min: int = 5000) -> tasks.Task:
+            n_steps_min: int = 5000,
+            compressed_trajectory: bool = False) -> tasks.Task:
```

Comment:
Maybe I would give other name to the variable. Because this will always be calculated. Maybe the variable could be download_full_trajectory and be set to false by default.